Modificaciones al README en español.

tammypmc opened the pull request:
Completa el issue #1

Commit:
Modificaciones al README en español. Arreglos ortograficos, indicaciones sobre latest.planet, arreglo de problema de guiones que no permitian ejecutar los step3 y step4

jamescr commented on README.md:
````diff
@@ -89,7 +87,7 @@ Se genera el fichero 'list_genderize.csv' como resultado.
 
 ```
-npm run step3 -- –-ciudad=nombreciudad
+npm run step3 -- --ciudad=nombreciudad
````
¿no me queda claro cuál es el cambio acá?

tammypmc replied:
El guión original venia en un formato que cuando se copiaba a la terminal se veía diferente a un guión normal y la terminal no reconocía el step por esta razon. Borrándolo y poniéndole un guión nuevo se arreglaba el problema.

jamescr commented on README.md:
````diff
@@ -127,7 +125,7 @@ Guardar el fichero corregido en la misma carpeta del proyecto, con el nombre:
 
 ```
-npm run step4 -- –-ciudad=nombreciudad
+npm run step4 -- --ciudad=nombreciudad
````
¿no me queda claro cuál es el cambio acá?

tammypmc replied:
El guión original venia en un formato que cuando se copiaba a la terminal se veía diferente a un guión normal y la terminal no reconocía el step por esta razon. Borrándolo y poniéndole un guión nuevo se arreglaba el problema.

jamescr commented:
Ok. Ya entiendo. Buenísimo. Quizás, para la próxima, sería bueno indicar el problema del guión en la descripción del PR.

Commit:
Se hacen los siguientes cambios al README en inglés:
 - Se traducen algunos segmentos de texto que permanecian en español
 - Se añade la aclaración para cuando se deba cambiar el nombre del archivo descargado a "latest.planet.mbtiles"
 - Se hacen arreglos estéticos en markdown
 - Se hace la corrección a los steps 3 y 4 donde uno de los guiones estaba en un formato diferente que a la hora de copiar y pegar en la terminal no reconocía el step
 - Se hacen correcciones de gramática y ortografía
 - Se cambian algunos links para que dirigan hacia las versiones en inglés de las páginas
